src/capture/capture.cpp

```diff
@@ -657,21 +1001,131 @@ void Capture::appendActions( std::string modelNumber, VRData::Hand hand )
+      }
+      else if ( input.second.type == "skeleton" )
+      {
+        // TODO: determine what to do with such an input!
+      }
+      else if ( input.second.type == "trackpad" )
+      {
+        if ( input.second.click )
+        {
+          actions.push_back( { VRData::ActionType::BOOLEAN, name, "click", hand } );
+        }
+        if ( input.second.force )
+        {
+          actions.push_back( { VRData::ActionType::SCALAR, name, "force", hand } );
+        }
+        // position and touch are automatic
+        actions.push_back( { VRData::ActionType::VECTOR2D, name, "position", hand } );
+        actions.push_back( { VRData::ActionType::BOOLEAN, name, "touch", hand } );
+        if ( input.second.value )
+        {
+          throw std::runtime_error(
+            std::format( "Encountered unsupported component <value> for input type <{}> named <{}>", input.second.type.c_str(), name.c_str() ) );
+        }
+      }
+      else if ( input.second.type == "trigger" )
+      {
+        if ( input.second.click )
+        {
+          actions.push_back( { VRData::ActionType::BOOLEAN, name, "click", hand } );
+        }
+        if ( input.second.force )
+        {
+          actions.push_back( { VRData::ActionType::SCALAR, name, "force", hand } );
+        }
+        if ( input.second.touch )
+        {
+          actions.push_back( { VRData::ActionType::BOOLEAN, name, "touch", hand } );
+        }
+        // value is automatic
+        actions.push_back( { VRData::ActionType::SCALAR, name, "value", hand } );
+      }
+      else
+      {
+        throw std::runtime_error( std::format( "Encountered unknown input type <{}> in file <{}>", input.second.type, m_profilePath ) );
+      }
+    }
+  }
+  return actions;
+}
+
 VRData::Device Capture::collectDeviceProperties( const vr::TrackedDeviceIndex_t deviceId ) const
 {
   VRData::Device device;
   device.m_deviceClass = m_vrSystem->GetTrackedDeviceClass( deviceId );
 
-  // We're not interested in Prop_GraphicsAdapterLuid_Uint64 and Prop_Invalid
-  // I don't know anything about Prop_ParentContainer
+  // TODO: don't know anything about Prop_Invalid or Prop_ParentContainer
   // TODO: don't know how to get binary data
   // TODO: don't know how to get vector3 data
   std::set<vr::ETrackedDeviceProperty> skippedProperties{ vr::ETrackedDeviceProperty::Prop_DisplayColorMultLeft_Vector3,
                                                           vr::ETrackedDeviceProperty::Prop_DisplayColorMultRight_Vector3,
                                                           vr::ETrackedDeviceProperty::Prop_DisplayHiddenArea_Binary_End,
                                                           vr::ETrackedDeviceProperty::Prop_DisplayHiddenArea_Binary_Start,
                                                           vr::ETrackedDeviceProperty::Prop_DisplayMCImageData_Binary,
-                                                          vr::ETrackedDeviceProperty::Prop_GraphicsAdapterLuid_Uint64,
```

Comment:
Does that mean we're not skipping this anymore?
Needs to be skipped, it records the adapter LUID that the app ran when capturing and can cause issues on replay

Reply:
Skipping it again.